src/plone/base/interfaces/controlpanel.py

```diff
@@ -57,13 +60,55 @@ def dump_json_to_text(obj):
 Disallow: /*view$
 """
 
+IMAGE_SRCSET_SCHEMA = json.dumps(
```

Comment:
This isn't exactly the most beautiful thing to see...

Reply:
Why? B/c it is on import time?

Reply:
Do we need this variable at all? 
This is just a string, why not having it the `image_srcsets` field?

Reply:
I think it would look ugly in the field definition because there is so much indentation. Calling `black` on it could even make it worse due to the line length. And we then have two big dictionaries within the field.